ios/LNDR/Info.plist

```diff
@@ -17,11 +17,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.40</string>
+	<string>1.41</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>46</string>
+	<string>47</string>
```

Comment:
Should this be Bundle Version 48 now?

Reply:
It will be 48 on the 1.5 update... right now, it's still 47 (1.4.1)